src/sentry/plugins/bases/notify.py

```diff
@@ -66,8 +68,15 @@ def get_plugin_type(self):
 
     def notify(self, notification):
         event = notification.event
-        return self.notify_users(event.group, event, triggering_rules=[
-                                 r.label for r in notification.rules])
+        try:
+            return self.notify_users(event.group, event, triggering_rules=[
+                                     r.label for r in notification.rules])
+        except requests.HTTPError as err:
+            self.logger.info('notification-plugin.notify-failed.', extra={
```

Comment:
Do we want to maybe have some metrics around errors? It would be good to be able to see if something is just completely broken, instead of swallowing it up and just logging

Reply:
What kind of metrics did you have in mind? With logging we could build a dashboard in kibana. I've started one for integrations and was considering adding this to it.